src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,42 @@
+package duke.task;
+
+import java.io.Serializable;
+import java.time.format.DateTimeFormatter;
+
+/** Class defining the minimal methods required to be implemented by a Task entity */
+public abstract class Task implements Serializable {
+    
+    /** Serializable classes must include a serialVersionUID to identify the class. */
+    private static final long serialVersionUID = 1561807677731348300L;
```

Comment:
Please abide by the naming convention for final variables (e.g. UI_LOGO).

Reply:
This is out of my control, as it is a field defined by the abstract class which I am implementing.